react/react2/week1/homework/user-search/src/components/SearchContext.js

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react";
+
+export const SearchContext = createContext();
+export const SearchContextProvider = SearchContext.Provider;
```

Comment:
All the context data(fetched data, loading, error, etc) can come in this file. then you can pass that data as values to children defined in App.js file.

Reply:
When there is more than 1 export function, I suggest making one export default.
Here SearchContext would be the default export.